### PR TITLE
Add Fluent Bit build_info metric

### PR DESCRIPTION
**Pull request by zhenyami**
Build info includes the release version tag, which is helpful when monitored hosts have different Fluent Bit versions installed.

* Exposed Fluent Bit `build_info` metric, which includes version.
* Added `DDEV_SKIP_GENERIC_TAGS_CHECK` to skip Fluent Bit's generic `version` tag validation.
* Updated tests.
* Synced `conf.yaml.example` file changes.

### What does this PR do?
Fluent Bit exposes its release version under `fluentbit_build_info` metric, although it's not [documented](https://docs.fluentbit.io/manual/administration/monitoring). This PR configures Datadog Fluent Bit integration to collect this metric.

### Motivation
Having this is useful for tracking Fluent Bit deployment changes, especially when the `version` tag is submitted with other Fluent Bit metrics via `share_labels` YAML config.

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [x] Feature or bugfix has tests
- [x] Git history is clean
- [x] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes
Additionally tested E2E – packaged the updated integration, uploaded and installed to the Agent on a host, and then validated metrics in the Datadog UI.

### CODE REVIEW

**Comment by zhenyami** [comment]
@DataDog/agent-integrations ready for review.

**Comment by zhenyami** [comment]
Hey @hithwen 👋
You helped me with this integration last year. Could you please help me find someone who can review this PR?
Thanks!

**Comment by zhenyami** [comment]
Hey @alopezz 👋 
Could you please help find a reviewer for this PR? Thank you!

File: fluentbit/metadata.csv [hunk 1]
```diff
@@ -9,4 +9,5 @@ fluentbit.output.errors.count,count,,error,,Number of errors in the output.,1,fl
 fluentbit.output.retries.count,count,,attempt,,Number of retries in the output.,1,fluent_bit,out_retry,
 fluentbit.output.retries_failed.count,count,,error,,Number of abandoned batches because the maximum number of retries was reached.,1,fluent_bit,out_retry_failed,
 fluentbit.output.retried_records.count,count,,entry,,Number of records retried by the output.,1,fluent_bit,out_retry_rec,
-fluentbit.output.dropped_records.count,count,,entry,,Number of records dropped by the output.,1,fluent_bit,out_drop_rec,
\ No newline at end of file
+fluentbit.output.dropped_records.count,count,,entry,,Number of records dropped by the output.,1,fluent_bit,out_drop_rec,
+fluentbit.build_info,gauge,,,,Release version and type,1,fluent_bit,build_info,
```
[the remark this answers is not in the document]
Comment (zhenyami) [reply fluentbit/metadata.csv]: Thanks for the suggestion. I updated the description.

**Review by steveny91 — APPROVED** [review]

**Comment by steveny91** [comment]
@zhenyami Looks good to me. Let me know if we're good to merge.

**Comment by zhenyami** [comment]
> @zhenyami Looks good to me. Let me know if we're good to merge.

Thank you, @steveny91. Please go ahead and merge.

**Pull request merged**